README.md

```diff
@@ -57,20 +53,26 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 Hello Rails Backend <a name="about-project"></a>
+# 📖 [Hello Rails Backend] <a name="about-project"></a>
+![hello-gif](https://thumbs.gfycat.com/PepperyGrizzledClownanemonefish-max-1mb.gif)
 
-This project is to implement a connection between a Ruby on Rails back-end and React front-end.
+**[Hello Rails Backend]** is a project that serves as the Backend for to display a random message, that will be consumed by the React app
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stacks <a name="tech-stack"></a>
+### Tech Stack <a name="tech-stack"></a>
 
-<!-- > Describe the tech stack and include only the relevant sections that apply to your project. -->
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
```

Comment:
* Kindly ensure that your Frontend renders random greetings as currently, the API works fine but the frontend doesn't render the expected results
![image](https://user-images.githubusercontent.com/52497246/211143972-9da5c909-278f-4632-aa19-ba1531fcd6b8.png)
![image](https://user-images.githubusercontent.com/52497246/211143977-5741a049-1dfb-4480-ba14-b57b18ba482f.png)